### ISSUE TITLE
Results on DOTA

### ISSUE THREAD

**Issue by MaFuyan** [LUCKMOONLIGHT/SLRDet#2]
Thanks for your awesome work. Your work is reallllllly helpful to me.
I did several experiments based on your project, but the results were not quite good. By the way, would you mind releasing the results on DOTA dataset?

**Comment by LUCKMOONLIGHT** [comment]
I feel very lucky to be able to help you.this project is just the basis of my work on remotesensing images includes DOTA dataset.I suggest you make some improvements according to the basis results and i am doing this too.I will releasing the results on DOTA dataset after a while.

**Comment by MaFuyan** [comment]
I really appreciate it that you could make your work open-sourced. Look forward to your brilliant paper. Thank you for your suggestions.

**Comment by qgh1223** [comment]
Can you give the code converting four points coordinate to bbox content?

**Comment by MaFuyan** [comment]
> Can you give the code converting four points coordinate to bbox content?

You can simply use cv2.minAreaRect(Points) to get what you want.

**Comment by qgh1223** [comment]
Why the angle is all smaller than 0.The source in your repository rs_devkit is available?Another problem is that why don't load pretrained resnet model weight and directly load weight from faster rcnn.It is so strange that training in other projects using mmdetection is not like this.The loss cannot convergence in a plane detection problem when I directly use minAreaRect function and cannot convergence if I load resnet weight and don't load faster rcnn weight.The loss is very big always bigger than 1.

**Comment by qgh1223** [comment]
bbox content like DOTA2COCOROTATED.py in you rs_devkit project?

**Comment by LUCKMOONLIGHT** [comment]
All code is tested and works.You can refer README.md to experiment.Data preprocess and postprocess script is in rs_devkit.
